app/api_clients/github.js

```diff
@@ -0,0 +1,42 @@
+const config = require('../config');
+const GithubAPI = require('github');
+const { ManagementClient } = require('auth0');
```

Comment:
I was wondering if we should keep the same convention we have in python with the order/grouping of requires.

Just an idea but it could help avoid confusion when we have 3rd party library vs. app-specific modules with similar names (e.g. `github` here is not one of our module, to not confuse with things like our `api_clients/github` module).

Obviously **no need to change anything**.

Reply:
Yeah, we should